### ISSUE TITLE
Image Block: Improve "Alt text" attribute description.

### ISSUE THREAD

**Issue by priethor** [WordPress/gutenberg#30872]
## What problem does this address?
The sidebar option "Alt text" asks the user to describe the purpose of the image or to leave it empty, which can result in users not knowing the importance of the alternative text to leave it empty.
![image](https://user-images.githubusercontent.com/27339341/114862525-ee41d080-9dee-11eb-9025-ba074e1a8db1.png)

## What is your proposed solution?
As proposed during [the FSE Go/No-go demo on April 14th](https://make.wordpress.org/core/2021/04/15/full-site-editing-go-no-go-april-14-2021/), the description should better explain why an alternative text is recommended in terms of accessibility and SEO.

**Comment by vdwijngaert** [comment]
FYI, this looks like it's identical to the WP Core Media Library.

![image](https://user-images.githubusercontent.com/772137/114882473-46370200-9e04-11eb-9e4c-07f959e51815.png)

This being said, I'm not sure that this description is the best place to convey importance of an alt text. A better place would be the documentation. Is there a guideline somewhere detailing what should and shouldn't be in a control's `help` property?

I'm not even sure about the impression of the [alt text decision tree](https://www.w3.org/WAI/tutorials/images/decision-tree/) it currently links to. Doesn't it look a bit overwhelming to the average user?

**Comment by JustinyAhin** [comment]
I agree with @vdwijngaert about having this info somewhere in documentation. Maybe here?  [Alt Text, alternative text for images](https://make.wordpress.org/accessibility/handbook/content/alternative-text-for-images/)

**Comment by priethor** [comment]
I also agree it should link to wordpress.org documentation rather than relying on external references.

**Comment by carolinan** [comment]
> I agree with @vdwijngaert about having this info somewhere in documentation. Maybe here? [Alt Text, alternative text for images](https://make.wordpress.org/accessibility/handbook/content/alternative-text-for-images/)

This page contains code examples and asks the reader to use CSS to insert images, it is not end user documentation, and in that sense the decision tree in the current link is easier to follow.
It also has an image of a screen that is not from the block editor, that can also be confusing.

**Comment by skorasaurus** [comment]
Hi, 

Thanks for reporting; 

This has been reported and discussed in https://github.com/WordPress/gutenberg/issues/19068. (as well as the https://core.trac.wordpress.org/ticket/48939 and https://github.com/WordPress/gutenberg/pull/26520
Feel free to [add your reaction](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-conversations-on-github#reacting-to-ideas-in-comments), discuss more, and follow its progress in the existing issues.
